app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -79,11 +79,10 @@
     <%= stylesheet_link_tag "application", media: "all" %>
     <!--[if lt IE 9]><%= javascript_include_tag "govuk_publishing_components/ie", integrity: true, crossorigin: "anonymous" %><![endif]-->
     <link rel="shortcut icon" href="<%= asset_path 'favicon.ico' %>" type="image/x-icon" />
-    <link rel="mask-icon" href="<%= asset_path 'govuk-mask-icon.svg' %>" color="#0b0c0c">
-    <link rel="apple-touch-icon" sizes="180x180" href="<%= asset_path "govuk-apple-touch-icon-180x180.png" %>">
-    <link rel="apple-touch-icon" sizes="167x167" href="<%= asset_path "govuk-apple-touch-icon-167x167.png" %>">
-    <link rel="apple-touch-icon" sizes="152x152" href="<%= asset_path "govuk-apple-touch-icon-152x152.png" %>">
-    <link rel="apple-touch-icon" href="<%= asset_path "govuk-apple-touch-icon.png" %>">
+    <link rel="mask-icon" href="<%= asset_path 'govuk-icon-mask.svg' %>" color="#0b0c0c">
+    <link rel="apple-touch-icon" sizes="180x180" href="<%= asset_path "govuk-icon-180.png" %>">
+    <link rel="apple-touch-icon" sizes="167x167" href="<%= asset_path "govuk-icon-192.png" %>">
+    <link rel="apple-touch-icon" sizes="152x152" href="<%= asset_path "govuk-icon-512.png" %>">
```

Comment:
Ideally drop these lines and add `manifest.json` if you can:

```suggestion
```

<br>

The `sizes` attributes don't match up as we've replaced older Apple-specific icon sizes:

* 167×167 @1x — Home screen (iPad Pro)
* 152×152 @1x — Home screen (iPad, iPad mini)

With more [generic "maskable" icons in **manifest.json**](https://web.dev/articles/maskable-icon) with different sizes:

* 180×180
* 192×192
* 512x512

You'll see the web manifest in action when adding to home screens or the Dock

<img src="https://github.com/alphagov/govuk_publishing_components/assets/415517/ef58274e-66c7-45ba-acb9-b37a9ef423ac" alt="Add to Dock">